pkgs/by-name/gi/gitoxide/package.nix

```diff
@@ -61,5 +61,8 @@ rustPlatform.buildRustPackage (finalAttrs: {
       asl20
     ];
     maintainers = with lib.maintainers; [ syberant ];
+    # NB: `ein` is also provided by this package, but `nix run
+    # nixpkgs#gitoxide` doesn't work at all without this set.
+    mainProgram = "gix";
```

Comment:
Usually when there's more than one binary provided, adding a `meta.mainProgram` is usually not advised.

How about using: `(lib.getExe' gitoxide "gix")` ?

Reply:
Thank you, setting this as `resolved` then.